src/geometry/manifold/times_cont_mdiff.lean

```diff
@@ -834,23 +904,43 @@ begin
   exact times_cont_diff_within_at.continuous_within_at
 end
 
+lemma times_cont_diff_within_at.times_cont_mdiff_within_at {f : E → E'} {s : set E} {x : E}
+  (hf : times_cont_diff_within_at 𝕜 n f s x) :
+  times_cont_mdiff_within_at (model_with_corners_self 𝕜 E) (model_with_corners_self 𝕜 E') n f s x
```

Comment:
There are still a lot of `model_with_corners_self` here and below.

Reply:
Oops, thanks!